Mesh_3/include/CGAL/internal/Mesh_3/helpers.h

```diff
@@ -45,7 +45,7 @@ void dump_graph_edges(std::ostream& out, const Graph& g)
   typedef typename boost::graph_traits<Graph>::edge_descriptor edge_descriptor;
 
   out.precision(17);
-  for(edge_descriptor e : edges(g))
+  for(edge_descriptor e : CGAL::make_range(edges(g)))
```

Comment:
Can you explain why this needed.  And why only here.

Reply:
`g` is a graph class from boost so edges is a `std::pair`. It is not tested by the testsuite and only activated with debug macros.